source/ways-of-working/environments/aks-auto-shutdown.html.md.erb

```diff
@@ -0,0 +1,57 @@
+---
+title: Environment schedule
+last_reviewed_on: 2023-02-07
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+### Schedule
+
+See below the list of environments that are shut down outside of working hours:
+
+| Shared Services | CFT | Shutdown | Start |
+|-|-|-|-|
+| Demo | Demo | 8pm | 7am |
+| ITHC | ITHC | 8pm | 7am |
+| Test | Perftest | 8pm | 7am |
+| PTLSBOX | PTLSBOX | 8pm | 7am |
+| Sandbox | Sandbox | 8pm | 7am |
```

Comment:
Thought it would be clearer divide the schedule column into shutdown and start. Your thoughts @timja?

Reply:
sure sounds good